### ISSUE TITLE
Formatting of change types on command line

### ISSUE THREAD

**Issue by ecraig12345** [microsoft/beachball#1]
Right now the list of change types on the command line is a little hard to parse visually: 

![image](https://user-images.githubusercontent.com/5864305/60537750-0a2dbe00-9cbe-11e9-9614-0d5483ff0ca2.png)

I think it would help to make the change type names ("Patch" etc) bold and/or change the spacing or punctuation.

I don't have a good way to simulate adding bold, but here are some ideas for changes to spacing/punctuation:

```
❯  Patch - bug fixes; no backwards incompatible changes.
   Minor - small feature; backwards compatible changes.
   None  - this change does not affect the published package in any way.
   Major - major feature; breaking changes.


❯  Patch  - bug fixes; no backwards incompatible changes.
   Minor  - small feature; backwards compatible changes.
   None   - this change does not affect the published package in any way.
   Major  - major feature; breaking changes.


❯  Patch: bug fixes; no backwards incompatible changes.
   Minor: small feature; backwards compatible changes.
   None:  this change does not affect the published package in any way.
   Major: major feature; breaking changes.


❯  Patch:  bug fixes; no backwards incompatible changes.
   Minor:  small feature; backwards compatible changes.
   None:   this change does not affect the published package in any way.
   Major:  major feature; breaking changes.


❯  Patch    bug fixes; no backwards incompatible changes.
   Minor    small feature; backwards compatible changes.
   None     this change does not affect the published package in any way.
   Major    major feature; breaking changes.


❯  Patch  (bug fixes; no backwards incompatible changes.)
   Minor  (small feature; backwards compatible changes.)
   None   (this change does not affect the published package in any way.)
   Major  (major feature; breaking changes.)
```

<img src="https://media.giphy.com/media/4J6A7l3xnv8lO/giphy.gif" width=300 />

**Comment by kenotron** [comment]
awesome gif!!
